### PR TITLE
Feature/cleanup

### PR DESCRIPTION
**Pull request by mitchellolsthoorn**

### CODE REVIEW

File: backend/src/main/nl/tudelft/ti2806/riverrush/controller/UserController.java [hunk 1]
```diff
@@ -44,8 +44,9 @@ public UserController(final EventDispatcher aDispatcher,
     public void initialize() {
         final HandlerLambda<JoinTeamCommand> joinTeamHandler = this::joinTeamHandler;
         final HandlerLambda<Event> sendOverNetworkLambda = (e) -> {
-            if (Objects.equals(e.getAnimal(), this.animal.getId())
-                    || Objects.equals(e.getAnimal(), -1)) {
+            Integer animalId = e.getAnimal();
+            if (Objects.equals(animalId, this.animal.getId())
```
Comment (txsmith) [inline backend/src/main/nl/tudelft/ti2806/riverrush/controller/UserController.java]: What's the reason for using Objects.equals here?

File: backend/src/main/nl/tudelft/ti2806/riverrush/game/state/WaitingGameState.java [hunk 2]
```diff
@@ -21,31 +25,36 @@
     private boolean readyToPlay = false;
 
     private final EventDispatcher dispatcher;
-    private Game game;
-    HandlerLambda<AnimalAddedEvent> AnimalAddedEventHandlerLambda = (e) -> animalAddedHandler();
+    private final Game game;
+    HandlerLambda<AnimalAddedEvent> AnimalAddedEventHandlerLambda = (e) -> checkForEnoughPlayers();
```
Comment (txsmith) [inline backend/src/main/nl/tudelft/ti2806/riverrush/game/state/WaitingGameState.java]: Variable names should not be begin in uppercase. Java conventions.

**Comment by newnottakenname** [comment]
@mitchellolsthoorn did you fix this yet?

**Comment by mitchellolsthoorn** [comment]
@newnottakenname no not yet, I'm going to do it now

**Comment by mitchellolsthoorn** [comment]
Fixed it now

**Comment by mitchellolsthoorn** [comment]
@newnottakenname Should I do a rebase, because there're a lot of merge conflicts?

**Pull request merged**